modules/olm-overriding-operatorconditions.adoc

```diff
@@ -51,4 +77,9 @@ spec:
     message: "The operator is performing a migration."
     lastTransitionTime: "2020-08-24T23:15:55Z"
 ----
+ifndef::openshift-dedicated,openshift-rosa[]
 <1> Allows the cluster administrator to change the upgrade readiness to `True`.
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+<1> Allows the `dedicated-admin` administrator to change the upgrade readiness to `True`.
+endif::openshift-dedicated,openshift-rosa[]
```

Comment:
I don't see _`dedicated-admin` administrator_ in the docs.

s/`dedicated-admin` administrator /`dedicated-admin` user